config/routes.rb

```diff
@@ -4,6 +4,8 @@
   namespace :api do
     post 'login', to: 'users#login'
     get 'feed', to: 'scores#user_feed'
-    resources :scores, only: %i[create destroy]
```

Comment:
Is there a reason we removed the `create` and `destroy` functionalities for scores?